Modules/HR/Routes/web.php

```diff
@@ -55,6 +55,7 @@
             Route::get('Dailyapplicationcount', 'ReportsController@index')->name('recruitment.reports.index');
             Route::get('reportsCard', 'ReportsController@showReportCard')->name('recruitment.daily-applications-count');
             Route::get('rejected-applications', 'ReportsController@rejectedApplications')->name('recruitment.rejected-applications');
+            Route::get('applications-rounds-duration', 'ReportsController@ApplicationRoundsDuration')->name('recruitment.application-rounds');
```

Comment:
This way it is easier to understand what this route is doing. It is a good practice to keep the naming convention such that the name of the variable/route/controller, etc. defines its purpose
```suggestion
            Route::get('reports/applications-rounds-duration', 'ReportsController@ApplicationRoundsDuration')->name('recruitment.reports.application-rounds');
```